### PR TITLE
update detection report creation method

### PR DESCRIPTION
**Pull request by yoouung**
- fix minor bugs about modules
- move html processing code from backend to frontend

### CODE REVIEW

**Comment by yoouung** [comment]
image

**Comment by yoouung** [comment]
pdf seems:
image

**Comment by doorooful** [comment]
> pdf seems: image

I think `검출모듈정보` part shows unexpected result. Can you fix on this PR?

File: dt-demo-be/app_lab.py [hunk 1]
```diff
@@ -10,6 +10,11 @@
 did = 1
 test_doc = Document(app, did)
 
+
+@app.route("/api/filter/modules", methods=["POST"])
+def post_selected_modules():
```
Comment (doorooful) [inline dt-demo-be/app_lab.py:15]: Please add TODO statement under this function

**Review by doorooful — CHANGES_REQUESTED** [review]
Please find the comment, and confirm why `검출모듈정보` part and `검출건수` table is showing different module names.

**Comment by yoouung** [comment]
> > pdf seems: image
> 
> I think `검출모듈정보` part shows unexpected result. Can you fix on this PR?

updated version !
image

**Review by doorooful — APPROVED** [review]
LGTM Thanks ◡̈

**Pull request merged**